helper/src/main/java/me/lucko/helper/menu/Gui.java

```diff
@@ -322,7 +326,7 @@ private void startListening() {
 
         Events.subscribe(InventoryDragEvent.class)
                 .filter(e -> e.getInventory().getHolder() != null)
-                .filter(e -> e.getInventory().getHolder().equals(this.player))
+                .filter(e -> Objects.equals(e.getInventory().getHolder(), this.player))
```

Comment:
If we're going to be replacing this equality check, it makes the most sense to straight up use a `this == that` qualification.

Reply:
according to IntelliJ, this is better

Reply:
Actually, IntelliJ thought that `getHolder()` would still be not-null even after the null check because it doesn't know about the preceding `filter`, so it gives you a null-safe `Object.equals` which is not required there.

I guess you can either revert this safely or remove the `!= null` check and use the null-safe `Object.equals` .